src/utils/index.js

```diff
@@ -41,8 +42,10 @@ export const parseFiltersToURL = (filters) => Object.keys(filters)
 export const ValidationReasons = Object.freeze({
```

Comment:
These are different from the ones in be, let's rename to `HumanValidationReasons`